components/google_cloud/actions/upload-object/upload-object.mjs

```diff
@@ -0,0 +1,42 @@
+import googleCloud from "../../google_cloud.app.mjs";
+
+export default {
+  name: "Upload An Object",
+  version: "0.0.1",
+  key: "google_cloud-upload-object",
+  type: "action",
+  description: "Upload an object to a GCS bucket",
```

Comment:
Could you please add a link to the docs and add the meaning of `GCS` here?

Reply:
done